helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -67,14 +67,16 @@ public enum InstanceConfigProperty {
     DELAY_REBALANCE_ENABLED,
     MAX_CONCURRENT_TASK,
     INSTANCE_INFO_MAP,
-    INSTANCE_CAPACITY_MAP, TARGET_TASK_THREAD_POOL_SIZE, HELIX_INSTANCE_OPERATIONS
+    INSTANCE_CAPACITY_MAP,
+    TARGET_TASK_THREAD_POOL_SIZE,
+    HELIX_INSTANCE_OPERATIONS
   }
 
   public static class InstanceOperation {
     private final Map<String, String> _properties;
 
     private enum InstanceOperationProperties {
-      OPERATION, REASON, SOURCE, TIMESTAMP
+      OPERATION, REASON, SOURCE, TIMESTAMP, LEGACY_DISABLED_TYPE
```

Comment:
Be consistent with formatting? I think I fixed Helix formatting in Helix-Style for intellij for this case. Please try it out.

If not, maybe fix it?